integration-tests/clipper_admin_tests.py

```diff
@@ -343,6 +345,20 @@ def predict_func(inputs):
             })
         self.assertEqual(len(containers), 1)
 
+    def test_test_predict_function(self):
+        def predict_func(xs):
+            return [sum(x) for x in xs]
+
+        deploy_python_closure(self.clipper_conn, name="sum-model", version=1, input_type="doubles", func=predict_func)
+        self.clipper_conn.link_model_to_app(app_name="hello-world", model_name="sum-model")
+        
+        headers = {"Content-type": "application/json"}
+        test_input = list(np.random.random(10))
+        pred = requests.post("http://localhost:1337/hello-world/predict", headers=headers, data=json.dumps({"input": test_input})).json()
+        test_predict_result = test_predict_function(self.clipper_conn, query={"input": test_input}, func=predict_func, input_type="doubles")
```

Comment:
You should call the function like this `self.clipper_conn.test_predict_function(query=...)`, not by passing in `self.clipper_conn` as an argument.


Can you add a second test prediction that uses the `batch_predict` interface? e.g.

```py
batch_input = [list(np.random.random(10)) for _ in range(4)]
batch_pred = requests.post("http://localhost:1337/hello-world/predict", headers=headers, data=json.dumps({"input_batch": batch_input})).json()
test_batch_predict_result = self.clipper_conn.test_predict_function(query={"input_batch": test_input}, func=predict_func, input_type="doubles")
```